dts/riscv/riscv32-fe310.dtsi

```diff
@@ -72,8 +72,10 @@
 			compatible = "sifive,gpio0";
 			gpio-controller;
 			interrupt-parent = <&plic>;
-			interrupts = <8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
-				26 27 28 29 30 31 32 33 34 35 36 37 38 39>;
+			interrupts = <8 1 9 1 10 1 11 1 12 1 13 1 14 1 15 1
```

Comment:
This would be more clear as `<8 1>, <9 1>, ...`.  Doing this has no effect on the generated array of phandles.

Applies to all instances of this property.

Reply:
Done. Thanks for review.